client/ayon_maya/plugins/publish/extract_pointcache.py

```diff
@@ -515,8 +519,23 @@ def get_attribute_defs(cls):
 
 
 class ExtractAnimation(ExtractAlembic):
-    label = "Extract Animation (Alembic)"
+    label = "Extract Animation"
     families = ["animation"]
+    optional = True
+    active = True
+
+    @classmethod
+    def apply_settings(cls, project_settings):
+        extract_animation_enabled = (
+            project_settings
+            ["maya"]
+            ["publish"].get("ExtractAnimation", {})
+        )
+        if extract_animation_enabled:
+            if not extract_animation_enabled.get("active"):
+                cls.active = False
+        else:
+            cls.enabled = False
```

Comment:
We do not transfer the optional state? Is that correct?

I feel like this logic is way more readable if we just apply the full `ExtractAnimation` as settings for this plug-in. Which I think is what is the intent?

Reply:
we dont need this...it's my bad, i forgot to remove